source/channels/archive-unarchive-channels.rst

```diff
@@ -4,12 +4,33 @@ Archive and unarchive channels
 .. include:: ../_static/badges/allplans-cloud-selfhosted.rst
   :start-after: :nosearch:
 
+.. |more-icon| image:: ../images/dots-horizontal_F01D8.svg
+    :alt: Use the More icon to access additional message options.
+
 Archive a channel
 -----------------
 
-Anyone can archive the public channels or private channels they belong to, unless the System Admin has restricted the permissions to do so. System Admins can archive channels in the System Console without needing to be a channel member. To archive a channel, select the channel name at the top of the center pane to access the drop-down menu, then select **Archive Channel**. 
+Anyone using Mattermost can archive the public channels or private channels they belong to, unless the system admin has restricted the permissions to do so. system admins can archive channels in the System Console without needing to be a channel member. When a channel is archived, it's removed from the user interface, but a copy exists on the server in case it's needed for audit reasons at a later stage. 
+
+.. note:: 
+  
+  From Mattermost v5.28, users can view, share, and search for archived channel content. Because a copy of the channel exists on the server, you can't reuse the URL of an archived channel when creating a new channel.
+
+.. tabs::
+
+  .. tab:: Web browser/desktop app
```

Comment:
I'd propose we simplify the tab labels on all associated pages to: "Desktop" "Mobile" 

End users don't need to consciously think about the differences between web and desktop app, especially since the behaviors is identical